src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/LanguageClient/RazorCustomMessageTargetTest.cs

```diff
@@ -266,7 +266,7 @@ async IAsyncEnumerable<ReinvocationResponse<IReadOnlyList<VSInternalCodeAction>>
 
         var documentSynchronizer = GetDocumentSynchronizer(GetCSharpSnapshot());
         var telemetryReporter = new Mock<ITelemetryReporter>(MockBehavior.Strict);
-        telemetryReporter.Setup(r => r.TrackLspRequest(It.IsAny<string>(), It.IsAny<string>(), It.IsAny<Guid>())).Returns(TelemetryScope.Null);
+        telemetryReporter.Setup(r => r.TrackLspRequest(It.IsAny<string>(), It.IsAny<string>(), TimeSpan.FromSeconds(1), It.IsAny<Guid>())).Returns(TelemetryScope.Null);
```

Comment:
Do you mean for these mocks to fail and need updating every time the threshold is changed? If not, switch to `It.IsAny<TimeSpan>()`.